modules/phase_field/src/userobjects/DiscreteNucleationMap.C

```diff
@@ -41,6 +41,11 @@ DiscreteNucleationMap::DiscreteNucleationMap(const InputParameters & parameters)
     _int_width(getParam<Real>("int_width")),
     _nucleus_list(_inserter.getNucleusList())
 {
+  // check for non-zero radius from old-style inputs and give error
+  if (!MooseUtils::absoluteFuzzyEqual(_radius, 0, libMesh::TOLERANCE * libMesh::TOLERANCE))
+    mooseError("Nucleus radius no longer specified in DiscreteNucleationMap.  See "
+               "DiscreteNucleationInserter to insert fixed or variable radius nuclei");
```

Comment:
please just remove the `radius` parameter!

Reply:
Can do.